C++/4. Advanced-Operators.md

```diff
@@ -116,11 +120,13 @@ Logical Operators are the operators which are used when comparing one more thing
 - `||`
 - `!`
 
-`&&` means and. \
-`||` means or. \
-`!` means not.
+`&&` means "and." \
+`||` means "or." \
+`!` means "not."
+
+The above operators are used with two Relational Operator statements or with two of other things. Like if you have `4 > 5 || 4 != 5` the result will be `true` because at least one of the two statements are true (`4 != 5`) and we said `||` so only one of them have to be true. `&&` means both / all of them have to be true and `!` means the result is reversed (`true` => `false` and `false` => `true`).
 
-The above operators are used with two Relational Operator statements or with two of other things. Like if you have `4 > 5 || 4 != 5` the result will be `1` because `1` stands for true and at least one of the two statements are true (`4 != 5`) and we said `||` so only one of them have to be true. `&&` means both / all of them have to be true and `!` means the result is reversed (`true` = `false` and `false` = `true`).
+**Note: In English, normally when you say that you're going to a store ___or___ a concert, you mean that you are intending to go to only one of the places, not neither, not both; C++'s `||` operator returns `true` for `true || true`, doesn't provide an operator that would yield `false` for `true || true`. (there is a way to capture the former meaning though, and will be covered later)**
```

Comment:
Don't think this is necessary

Reply:
I mean you're ranting at the end saying that C++ doesn't provide a certain operator isn't needed.

Reply:
It was intended to contrast human intuition against how C++ operators work, but I'll remove it.